index.html

```diff
@@ -74,6 +75,220 @@ <h1 class="main-text">I’m Giovanni Glad to see you!</h1>
+
+        </div>
+    </section>
+
+    <!-- Tonic Section -->
+    <section>
+        <div class="work-container">
+            <div class="work-container-background">
+                <div class="work-image-container">
+                    <img src="./Assets/mobile_Snapshoot_Portfolio_3.png" alt="">
+                </div>
+    
+                <div class="work-text-container">
+                    <h1>Tonic</h1>
+                    <div class="work-complementary-text">
+                        <ul class="work-complementary-list">
+                            <li class="canopy">CANOPY</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">Back End Dev</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div>
+                        <ul class="buttons-work-container">
+                            <li class="btn-work-style">html</li>
+                            <li class="btn-work-style">css</li>
+                            <li class="btn-work-style">Javascript</li>
+                        </ul>
+                    </div>
+                    <button class="button-work-project">SEE PROJECT</button>
+                </div>
+
+            </div>
+
+        </div>
+    </section>
+
+    <!-- Second Multi Post Section -->
+    <section>
+        <div class="work-container">
+            <div class="work-container-background">
+                <div class="work-image-container">
+                    <img src="./Assets/mobile_Snapshoot_ Portfolio_4.png" alt="">
+                </div>
+    
+                <div class="work-text-container">
+                    <h1>Tonic</h1>
+                    <div class="work-complementary-text">
+                        <ul class="work-complementary-list">
+                            <li class="canopy">CANOPY</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">Back End Dev</li>
+                            <li class="canopy-complement"><img src="./Assets/Counter.png" alt="">2015</li>
+                        </ul>
+                    </div>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div>
+                        <ul class="buttons-work-container">
+                            <li class="btn-work-style">html</li>
+                            <li class="btn-work-style">css</li>
+                            <li class="btn-work-style">Javascript</li>
+                        </ul>
+                    </div>
+                    <button class="button-work-project">SEE PROJECT</button>
+                </div>
+
+            </div>
+
+        </div>
+    </section>
```

Comment:
![Capture2](https://user-images.githubusercontent.com/93039359/198691049-bab29d26-fcb2-4363-95d7-163a8462526a.PNG)

- [ ] I know that responsiveness is not a part of your project for now but your project should expand on all the mobile screens. In the above image you can see that your application is not expanding. Kindly remove all the fix widths for your sections to fix this problem 👍